tests/phpunit/tests/rest-api/rest-block-type-controller.php

```diff
@@ -338,6 +338,67 @@ public function test_get_item_defaults() {
 		$this->assertNull( $data['style'] );
 	}
 
+	/**
+	 * @ticket 56733
+	 */
+	public function test_get_item_deprecated() {
+		$block_type = 'fake/deprecated';
+		$settings   = array(
+			'editor_script' => 'hello_world',
+			'script'        => 'gutenberg',
+			'view_script'   => 'foo_bar',
+			'editor_style'  => 'guten_tag',
+			'style'         => 'out_of_style',
+		);
+		register_block_type( $block_type, $settings );
+		wp_set_current_user( self::$admin_id );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/block-types/' . $block_type );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+		$this->assertSameSets( array( 'hello_world' ), $data['editor_script_handles'] );
+		$this->assertSameSets( array( 'gutenberg' ), $data['script_handles'] );
+		$this->assertSameSets( array( 'foo_bar' ), $data['view_script_handles'] );
+		$this->assertSameSets( array( 'guten_tag' ), $data['editor_style_handles'] );
+		$this->assertSameSets( array( 'out_of_style' ), $data['style_handles'] );
+		// Deprecated properties.
+		$this->assertSame( 'hello_world', $data['editor_script'] );
+		$this->assertSame( 'gutenberg', $data['script'] );
+		$this->assertSame( 'foo_bar', $data['view_script'] );
+		$this->assertSame( 'guten_tag', $data['editor_style'] );
+		$this->assertSame( 'out_of_style', $data['style'] );
+	}
+
+	/**
+	 * @ticket 56733
+	 */
+	public function test_get_item_deprecated_with_arrays() {
+		$block_type = 'fake/deprecated-with-arrays';
+		$settings   = array(
+			'editor_script' => array( 'hello', 'world' ),
+			'script'        => array( 'gutenberg' ),
+			'view_script'   => array( 'foo', 'bar' ),
+			'editor_style'  => array( 'guten', 'tag' ),
+			'style'         => array( 'out', 'of', 'style' ),
+		);
+		register_block_type( $block_type, $settings );
+		wp_set_current_user( self::$admin_id );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/block-types/' . $block_type );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+		$this->assertSameSets( $settings['editor_script'], $data['editor_script_handles'] );
+		$this->assertSameSets( $settings['script'], $data['script_handles'] );
+		$this->assertSameSets( $settings['view_script'], $data['view_script_handles'] );
+		$this->assertSameSets( $settings['editor_style'], $data['editor_style_handles'] );
+		$this->assertSameSets( $settings['style'], $data['style_handles'] );
+		// Deprecated properties.
+		// Since the schema only allows strings or null (but no arrays), we return the first array item.
+		$this->assertSame( 'hello', $data['editor_script'] );
+		$this->assertSame( 'gutenberg', $data['script'] );
+		$this->assertSame( 'foo', $data['view_script'] );
+		$this->assertSame( 'guten', $data['editor_style'] );
+		$this->assertSame( 'out', $data['style'] );
```

Comment:
Test methods with multiple assertions need a `message` parameter for each assertion, per [Core Handbook - Writing PHPUnit Tests - Using Assertions](https://make.wordpress.org/core/handbook/testing/writing-phpunit-tests/#using-assertions).

Reply:
Addressed in 9bc56a5 😊